docs/user-guide/install.md

```diff
@@ -6,18 +6,81 @@ vSphere Docker Volume Service is simple to install. It has zero configuration an
 
-<script type="text/javascript" src="https://asciinema.org/a/80405.js" id="asciicast-80405" async></script>
+```
+# esxcli software vib install -v /tmp/vmware-esx-vmdkops-0.12.ccfc38f.vib
+Installation Result
+   Message: Operation finished successfully.
+   Reboot Required: false
+   VIBs Installed: VMWare_bootbank_esx-vmdkops-service_0.12.ccfc38f-0.0.1
+   VIBs Removed:
+   VIBs Skipped:
+```
 
 # Install on VM
 
-We currently package the service as a RPM and Deb package. This is to be able to start the service before Docker engine starts. We will also support Docker plugin framework once it is ready.
+vDVS 0.13 comes with managed plugin support using Docker plugin framework and please note DEB/RPM packages will be deprecated going forward and will not be available.
 
+**For Docker 1.13 and above**, install managed plugin from [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description).
+```
+docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:latest
+```
+
+**Using vDVS managed plugin**
+
+* **Install vDVS plugin**
+```
+~# docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:latest
+0.12: Pulling from vmware/docker-volume-vsphere
+27de4a000433: Download complete
+Digest: sha256:696017e620fe8d29c883f36b5676135eaa2c2b3c3810f82931efb8027369a186
+Status: Downloaded newer image for vmware/docker-volume-vsphere:0.12
+Installed plugin vmware/docker-volume-vsphere:0.12
+```
+
+Let's verify the installed plugin using `docker plugin ls` command. You can see that vDVS plugin is not enabled yet. You have to enable first.
+```
+~# docker plugin ls
+ID                  NAME                DESCRIPTION                           ENABLED
+de30e65cda7b        vsphere:latest      VMWare vSphere Docker Volume plugin   false
+```
+
+* **Enable vDVS plugin** Using `docker plugin enable <plugin_name>`
+```
+~# docker plugin enable vsphere
+vsphere
+~# docker plugin ls
+ID                  NAME                DESCRIPTION                           ENABLED
+de30e65cda7b        vsphere:latest      VMWare vSphere Docker Volume plugin   true
+```
+
+* **Disable vDVS plugin** Using `docker plugin disable <plugin_name>`
+```
+~# docker plugin disable vsphere
+vsphere
+~# docker plugin ls
+ID                  NAME                DESCRIPTION                           ENABLED
+de30e65cda7b        vsphere:latest      VMWare vSphere Docker Volume plugin   false
+```
+
+* **Remove vDVS plugin** Using `docker plugin rm <plugin_name>`
+```
+~# docker plugin rm vsphere
+vsphere
+~# docker plugin ls
+ID                  NAME                DESCRIPTION         ENABLED
+```
+
+**For Docker 1.12 and earlier**, use DEB or RPM package.
```

Comment:
This is time being block and will be removed by next release so kept it as it is. 

Let me know what you think.